WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewController.swift

```diff
@@ -722,7 +727,10 @@ private extension DashboardViewController {
                                                                                    calendar: .current) { [weak self] isVisibleFromAppSettings in
                     guard let self = self else { return }
 
-                    let shouldShowJetpackBenefitsBanner = site?.isJetpackCPConnected == true && isVisibleFromAppSettings
+                    let isJetpackCPSite = site?.isJetpackCPConnected == true
```

Comment:
Super nit: Can `site?.isJetpackCPConnected == true` be shortened as just `site?.isJetpackCPConnected`?

Reply:
This would not work since `site?.isJetpackCPConnected` is an optional `bool` so we will need to either unwrap it or compare it to `true`.